RecoTauTag/RecoTau/plugins/DeepTauId.cc

```diff
@@ -158,7 +165,7 @@ float dEta(const LVector1& p4, const LVector2& tau_p4)
 template<typename LVector1, typename LVector2>
 float dPhi(const LVector1& p4, const LVector2& tau_p4)
 {
-    return static_cast<float>(ROOT::Math::VectorUtil::DeltaPhi(p4, tau_p4));
+    return static_cast<float>(reco::deltaPhi(p4.phi(), tau_p4.phi()));
```

Comment:
I think that one can use `reco::deltaPhi(p4, tau_p4)`, i.e. not call `phi()` method of four-vector.

Reply:
without calling `phi()` it does not compile, because it takes a wrong `deltaPhi` specialization: https://github.com/cms-sw/cmssw/blob/master/DataFormats/Math/interface/deltaPhi.h#L47-L50

Reply:
OK, thanks